assets/javascripts/modules/responsiveInlineDetails.js

```diff
@@ -0,0 +1,30 @@
+module.exports = function() {
+
+  $('[data-responsive-inline-details]').each(function() {
+    var $detailsHolder,
+    $detailsTargets,
+    $detailsButton;
```

Comment:
You can assign these at the time as declaring them. There's no benefit to declaring them on their own.